### ISSUE TITLE
Localisation  GPS Android MapBox unity  won't open

### ISSUE THREAD

**Issue by souleimagara** [mapbox/mapbox-unity-sdk#1565]
_Note: If this is a bug or support ticket, please provide the following information:_
* Unity version: 2019.1.1f1 
  * Scripting Runtime Version: 4.x
  * Scripting Backend: Mono
  * Api Compatibility Level: .NET 4.x
* Mapbox SDK version: v2.1.1
* The platform you're building to: Android
* A description of what you're trying to do:  Localisation GPS won't open

* Steps to recreate the bug if appropriate:
I import SDK map box package to my unity project and I'm using the Location-Based Game scene I create my own styled map box than I change the style name to custom and I coped my Own map URL,  then I build my application to android and i installed in my android phone  hen I run it I get this usual notification "Allow the application to enable GPS and use your location " I click Allow and nothing happing the GPS won't open I had to open it manually every time to use the application functionally I have no idea what could be wrong I, can't find any solution how to fix this bug  
* Links to your logs are also helpful:

**Comment by jordy-isaac** [comment]
Can you confirm you have a network connection or GPS signal? Have you made any changes to the example scene?

**Comment by souleimagara** [comment]
Yes, I confirm that I have a network connection and a GPS signal.
No, I haven't made any changes in the example Scene I just change the URL of my Styled Map and My token. I even reinstall the MapBox SDK and restart all over and still the same problem   
In the Location-Based Game Scene  after I build it at the text field, I get GPS Not Enable but it's enabled in my phone
